src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComActivator.cs

```diff
@@ -76,13 +80,21 @@ public struct ComActivationContextInternal
     {
         public Guid ClassId;
         public Guid InterfaceId;
+        public IntPtr AssemblyPathBuffer;
         public IntPtr AssemblyNameBuffer;
         public IntPtr TypeNameBuffer;
         public IntPtr ClassFactoryDest;
     }
 
     public static class ComActivator
     {
+        // Collection of all ALCs used for COM activation. In the event we want to support
+        // unloadable COM server ALCs, this will need to be changed.
+        // This collection is read-only and if new ALCs are added, the collection should
+        // be copied into a new collection and the static member updated.
+        private static Dictionary<string, AssemblyLoadContext> s_AssemblyLoadContexts = new Dictionary<string, AssemblyLoadContext>(StringComparer.CurrentCultureIgnoreCase);
```

Comment:
This should be InvariantCultureIngoreCase. The file system case insensitivity is not culture-specific.

Reply:
Will do.

>The file system case insensitivity is not culture-specific.

Didn't know that.